### PR TITLE
Update RoomScreen and UserSessionScreen UI tests

### PR DESCRIPTION
**Pull request by Velin92**

### CODE REVIEW

File: UITests/Sources/RoomScreenUITests.swift [hunk 1]
```diff
@@ -75,8 +75,7 @@ class RoomScreenUITests: XCTestCase {
         try await app.assertScreenshot(.roomSmallTimelineLargePagination)
     }
 
-    // This test is DISABLED because it's flakey on the CI
-    func disabled_testTimelineLayoutInMiddle() async throws {
+    func testTimelineLayoutInMiddle() async throws {
```
Comment (Velin92) [inline UITests/Sources/RoomScreenUITests.swift:78]: @pixlwave let's give this another chance? maybe in SwiftUI will work better?

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomSmallTimelineWithReactions.png
Comment (Velin92) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomSmallTimelineWithReactions.png:1]: What about this? @pixlwave 
I can't find the test for it in the code anywhere!

File: UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.roomLayoutBottom-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.roomLayoutBottom-1.png:1]: This one is missing the keyboard

File: UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.roomLayoutMiddle-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/en-GB-iPhone-14.roomLayoutMiddle-1.png:1]: And this one (although it was broken before by the looks of things)

File: UITests/Sources/__Snapshots__/Application/pseudo-iPhone-14.roomLayoutBottom-1.png
Comment (pixlwave) [inline UITests/Sources/__Snapshots__/Application/pseudo-iPhone-14.roomLayoutBottom-1.png:1]: Keyboard on this one too.

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomSmallTimelineWithReactions.png
Comment (Velin92) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomSmallTimelineWithReactions.png:1]: looked in all the project I can't find them, maybe they were removed somewhere by mistake?

File: UITests/Sources/RoomScreenUITests.swift (hunk 1 above)
Comment (pixlwave) [reply UITests/Sources/RoomScreenUITests.swift:78]: Yeah would be nice! I'm not sure the snapshots are doing what they should be though. I'll take a shot at fixing it tomorrow though

**Review by pixlwave — APPROVED** [review]
Couple of keyboards missing, otherwise lgtm

File: UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomSmallTimelineWithReactions.png
Comment (langleyd) [reply UITests/Sources/__Snapshots__/Application/en-GB-iPad-9th-generation.roomSmallTimelineWithReactions.png:1]: Oh I think I removed the test as the reactions were already rendered in another test. So we can probably remove these images?

**Comment by Velin92** [comment]
there is an issue with layoutBottom ones fixing it

**Pull request merged**